ledger-core/v2/virtual/execute/execute.go

```diff
@@ -294,23 +294,70 @@ func (s *SMExecute) stepTakeLock(ctx smachine.ExecutionContext) smachine.StateUp
 		return ctx.Sleep().ThenRepeat()
 	}
 
+	if s.Payload.CallRequestFlags.GetRepeatedCall() == payload.RepeatedCall {
+		return ctx.Jump(s.stepDeduplication)
+	}
+
 	return ctx.Jump(s.stepStartRequestProcessing)
 }
 
-func (s *SMExecute) stepStartRequestProcessing(ctx smachine.ExecutionContext) smachine.StateUpdate {
-	var objectDescriptor descriptor.Object
+func (s *SMExecute) stepDeduplication(ctx smachine.ExecutionContext) smachine.StateUpdate {
```

Comment:
`stepDeduplicate`, pls. `Deduplication` is process, `Deduplicate` is action

Reply:
fixed